docs/content/usage/badge.en-us.md

```diff
@@ -0,0 +1,36 @@
+---
+date: "2023-02-25T00:00:00+00:00"
+title: "Badge"
+slug: "badge"
+sidebar_position: 11
```

Comment:
What's `sidebar_position`? I just choose one number between the previous and next page. But I don't what it is and couldn't find the specification of it.

Reply:
The sequence under `Usage`.